### PR TITLE
kbp: fix team git site root

### PR DESCRIPTION
**Pull request by songgao**
Tested on staging and made sure it works for both team folder and team git.

### CODE REVIEW

File: libpages/root.go [hunk 1]
```diff
@@ -49,6 +49,19 @@ func (t RootType) String() string {
 	}
 }
 
+func tlfTypeToPath(t tlf.Type) string {
+	switch t {
+	case tlf.Private:
+		return string(libkbfs.PrivatePathType)
```
Comment (strib) [inline libpages/root.go]: Technically the "right" variables to use would be the ones in `libgit/autogit_node_wrappers.go", which define how the tlf types map to subdirs for autogit specifically.  (They would have to be exported.)

But this is fine too I guess, just a little dirtier.

Comment (songgao) [reply libpages/root.go]: Ah I only saw the ones in `libkbfs`. I'll export it from `libgit`!

**Comment by songgao** [comment]
@jzila `tlf.Type.String()` returns `singleTeam` for `tlf.SingleTeam`

**Comment by jzila** [comment]
Got it, thanks.

**Pull request merged**